library/kani/src/tuple.rs

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Modifications Copyright Kani Contributors
+// See GitHub history for details.
+
+//! Support for arbitrary tuples where each element implements
+//! `kani::Arbitrary`. Tuples of size up to 11 are supported in this
+//! file.
+
+use crate::Arbitrary;
+
+/// This macro implements `kani::Arbitrary` on a tuple whose elements
+/// already implement `kani::Arbitrary` by running `kani::any()` on
+/// each index of the tuple.
+macro_rules! tuple {
+    ($($typ:ident),*) => {
+	impl<$($typ : Arbitrary),*>  Arbitrary for ($($typ,)*) {
+            #[inline(always)]
```

Comment:
Is the indentation off on this line?

Reply:
72a48f243631eef863686f1726db395a867df4a0